vector/src/main/java/im/vector/app/core/utils/ExternalApplicationsUtil.kt

```diff
@@ -305,29 +305,21 @@ fun shareMedia(context: Context, file: File, mediaMimeType: String?) {
         return
     }
 
-    val sendIntent = ShareCompat.IntentBuilder(context)
+    val chooserIntent = ShareCompat.IntentBuilder(context)
             .setType(mediaMimeType)
             .setStream(mediaUri)
-            .getIntent()
-
-    sendShareIntent(context, sendIntent)
+            .setChooserTitle(R.string.share)
+            .createChooserIntent()
+    context.startActivity(chooserIntent)
```

Comment:
I would prefer to keep `ActivityNotFoundException` handling, since on some exotic devices it can still happen.

Reply:
:heavy_check_mark: Changed in 74363ff8235764c0dc15ebb6b3dce42a6f52b56d.